cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/orion/OrionBackendImpl.java

```diff
@@ -76,12 +76,7 @@ public JsonResponse deleteSubscriptionV1(String subscriptionId, String token)
         
         // create the http header
         ArrayList<Header> headers = new ArrayList<Header>();
```

Comment:
This (and other similar lines) could be within the `setHeaders()` method (which could be also renamed as `getHeaders()`:

    ArrayList<Header> = getHeaders(token);